Sourcery/Parsing/Parser.swift

```diff
@@ -375,6 +375,12 @@ final class Parser {
 
         for (_, type) in unique {
             if let enumeration = type as? Enum, enumeration.rawType == nil {
+                enumeration.cases.forEach { enumCase in
+                    enumCase.associatedValues.forEach { associatedValue in
+                        associatedValue.type = unique[associatedValue.typeName]
```

Comment:
Done in 0d82f3c.

I also added `Enum.Case.AssociatedValue.unwrappedTypeName` (same as `Variable`) for safely parsing optional type.
This could be abstracted in future update 😉